packages/civic-sandbox/src/state/fetch-adapter.js

```diff
@@ -1,19 +1,46 @@
 import axios from 'axios';
 
-const HOST = 'https://sandbox.civicpdx.org/civic-sandbox';
 const echo = a => a;
 
-const apiAdapter = (url, { encodeParams, start, success, failure }) => params => (dispatch) => {
+const apiAdapter = (url, { start, success, failure }) => () => (dispatch) => {
   dispatch(start());
-
-  const encode = encodeParams || echo;
-  const fullURL = encode(HOST + url, params);
-  return axios.get(fullURL).then((res) => {
+  return axios.get(url).then((res) => {
     dispatch(success(res.data));
     return res;
   }).catch((err) => {
     dispatch(failure(err));
   });
 };
 
+export const fetchByDateAdapter = (slide, date, type, { start, success, failure }) => () => (dispatch) => {
+  dispatch(start());
+  return axios.get(`${slide.endpoint}${date}`).then((res) => {
+    dispatch(success({
+      name: slide.name,
+      data: res.data,
+      type,
+    }));
+    return res;
+  }).catch((err) => {
+    dispatch(failure(err));
+  });
+};
+
+export const fetchAllSlidesAdapter = (slides, { start, success, failure }) => () => (dispatch) => {
+  dispatch(start());
+  const fullUrls = slides.map(slide => axios.get(slide.endpoint));
+  return axios.all(fullUrls).then(axios.spread((...res) => {
```

Comment:
Neat! I need to look more into the full axios API. It's got all sorts of stuff I've never seen.